### DISCUSSION TITLE
Playtime release candidate 2.16.15 is here!

### DISCUSSION THREAD

**Discussion by helgoboss** [helgoboss/helgobox#1357] [Announcements]
Please let me know if you experience any major issues with it 🙏

**Comment by Bytheburn** [comment]
Hiya Heloboss. Just submitted this message to Support - "MAJOR BUG? HELP: I was using Playtime 2 when the grid suddenly disappeaerd and now, whatever I do, I can't get it back. I can add colums but there are no rows and no way to create them so I'm comletely stymied and can do nothing.

How do I reset Playtime as whenever I create a new instance by whatever means I get a blank screen with only the Master and a + Button showing in the main panel. If I click the + a column is created and a track in Reaper but the message "Bad State: no element" is displayed.
FYI: I'm running updated Win 10 with no intention of vupgrading to 11 and the latest version of Reaper."

Just to add I'm a registered user and I had just updated Playtime via Reapack and everything was working fine for an hour then it wasn't, as described above. I've no idea what triggered the bug. Reaper was working as usual (no crash).

P.S. I noticed my Playtime 1, which I had been using. now crashes whenever I try to Record so I need Playtime 2 to work reliably asap.

Thanks in advance...

**Comment by Bytheburn** [comment]
P.P.S. Re previous msg this is acreen shot of what I get when I open playtime. There's no option to create a matrix. If I click the "+" I get tan error...
![image](https://github.com/user-attachments/assets/96127404-812e-41eb-aead-0ebcef05ca62)
![image](https://github.com/user-attachments/assets/ffecb17b-4667-4534-89a6-d5fce4c342b3)

**Reply by helgoboss** [reply]
Hi @Bytheburn. I can see that you are using a very old version: 2.16.0-pre.15. Could you try updating to the latest and try again (2.16.15)? Also, which REAPER version are you using?

**Reply by Bytheburn** [reply]
Hi Benjamin,
I assume you got my Resource Folder? I was surprised I wasn't running the latest version as I had explicitely updated via Reapack synchronisation immediately before the issue happened. I just synchromised again and this is the message I get:
![image](https://github.com/user-attachments/assets/79132a17-6948-4624-b861-74bff1b0ac57)

OK. I've just done a Reapack Browse and I'm seeing 2.16.15 so I'm explicitely uninstalling 2.16.0 - pre and installing2.16.15 right now. I'll get back to you after I've done that and tested it...

![image](https://github.com/user-attachments/assets/6689cd2b-5305-401c-85b7-12f06df3db9a)

**Reply by Bytheburn** [reply]
Quick update. When I uninstalled 2.16.0 Reapack said it had automatically installed 2,16.15 so I'm testing that now... 
![image](https://github.com/user-attachments/assets/e5804d55-4313-410d-b779-c2c707a347c7)

**Reply by Bytheburn** [reply]
Wrire 2.16.15 installed and restarted reaper and I get an error when I start Playtime from the taskbar button...

![image](https://github.com/user-attachments/assets/39c66a0d-839f-4155-8636-e308f607679d)

**Reply by Bytheburn** [reply]
I tried launching from the helgoboss menu and got the same error Clicking on the error box I get this screen so I'll try following those instructions...
![image](https://github.com/user-attachments/assets/2226a2a9-9674-4db4-a633-33e03e2dad7a)

**Reply by Bytheburn** [reply]
I tried to comply but I can't find anything like:

Options--> Settings--> Audio--> Playback--> Stop/repeat playback at end of project

...either on the Options menu or in Preferences. I'm probably being dumb but could you give further details? This is my Reaper version
![image](https://github.com/user-attachments/assets/9c7f83ca-ea3d-4811-a486-247b462e62d4)

**Reply by Bytheburn** [reply]
Okay I found it: 
![image](https://github.com/user-attachments/assets/788368b3-681c-40ea-af9e-8a2122f5711a)

I no longer get the error when I launch Playtime, but neither do I get any matrix or any option to create one. If I click "+" to add a channel the channel is created but no matrix or sell. 

I closed the window to see if that would cause it to update but when I click on FX to reopen it allI get now is a Realearn window:
![image](https://github.com/user-attachments/assets/cc075c76-4ffd-424d-b5fa-8bccd79525b2)

**Reply by Bytheburn** [reply]
Restarted Reaper. I'm now getting the "Bad State, No Element" error I was getting with the old version again: :-(

![image](https://github.com/user-attachments/assets/ccc1da99-60a6-4173-80f7-aacfa044ee9d)

I'll try reinstalling Playtime via Reapack again next. The only thing I can think of beyond that is to unistall Realearn/Playtime completely and then reinstall but I'm reuctant to do that as I multiple complex templates based on Realearn

Would uninstalling/reinstalling Relearn/Playtime break my existing setup?

I won't try that until I hear back from you that it's safe...

**Reply by Bytheburn** [reply]
Reinstalled Playtime via Reapack and I get just the same errors so I'm back where I started which is troubling :-(

All I can do now is uninstall then reinstall Realearn/Playtime completely but, as I stated above:

"I'm reluctant to do that as I multiple complex templates based on Realearn. Would uninstalling/reinstalling Relearn/Playtime break my existing setup? I won't try that until I hear back from you that it's safe..."

I await your reponse...

**Reply by helgoboss** [reply]
I will have a look into the ZIP that you sent me. Maybe this brings some insight.

**Reply by helgoboss** [reply]
Oh, and this deserved its own ticket: #1368. I will track progress there.

**Reply by Bytheburn** [reply]
Hi Benjamin,

I'm responding here just in case because you marked the bug report as 'closed' and the 'bot' may have archived it as done so I'm not sure if you will have got my last comment on the bug thread.

I think you've closed this prematurely (see below)...

Clicking in the boxes at the top right of the window I do indeed now have the matrix back and when I open a new instance I get the option to create a matrix. I can also confirm the "Bad State: No element" error only occurs when you create a new column when the matrix is hidden.

However, I've still got a serious problem because whenever I close the window and try to reopen it I end up back in a ReaLearn window with no way I can see to restore what I was working on and I've apparently lost all the data I had entered. Clearly this is 100% unusable!
![image](https://github.com/user-attachments/assets/bb6c9620-c2d5-4283-8df7-9dcb9cd3dd91)

**Reply by helgoboss** [reply]
I think I already responded to this one on another channel, but just for the logs: The app (including the Playtime GUI) can be opened via "Menu => Show app", when in ReaLearn. This is typically not necessary because there's the more convenient Playtime toolbar button. See the docs, thanks.
